src/microphysics/opac_phoebus/opac_phoebus.cpp

```diff
@@ -270,8 +273,11 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
       const Real lTMin = pin->GetOrAddReal("mean_opacity", "ltmin", std::log10(1.e5));
       const Real lTMax = pin->GetOrAddReal("mean_opacity", "ltmax", std::log10(1.e12));
       const int NT = pin->GetOrAddInteger("mean_opacity", "nt", 10);
-      MeanSOpacity mean_opac_host = MeanSOpacityCGS(opacity_host, lRhoMin, lRhoMax, NRho,
-                                                    lTMin, lTMax, NT, YeMin, YeMax, NYe);
+      auto cgs_mean_opacity = MeanSOpacityCGS(opacity_host, lRhoMin, lRhoMax, NRho, lTMin,
+                                              lTMax, NT, YeMin, YeMax, NYe);
+      MeanSOpacity mean_opac_host = MeanNonCGSUnitsS<MeanSOpacityCGS>(
+          std::forward<MeanSOpacityCGS>(cgs_mean_opacity), time_unit, mass_unit,
+          length_unit, temp_unit);
```

Comment:
This fixes a unit issue with mean opacities that was fixed in `singularity-opac`